client.lua

```diff
@@ -898,21 +897,44 @@ end)
 
 -- Stress Gain
 
+local function IsWhitelistedVehClass(vehClass)
+    if vehClass then
+        for _, v in pairs(Config.WhitelistedVehClasses) do
+            if vehClass == v then
+                return true
+            end
+        end
+    end
+    return false
+end
+
+local function IsWhitelistedVehicle(vehHash)
+    if vehHash then
+        for _, v in pairs(Config.WhitelistedVehicles) do
+            if vehHash == GetHashKey(v) then
+                return true
+            end
+        end
+    end
+    return false
+end
+
 CreateThread(function() -- Speeding
     while true do
         if LocalPlayer.state.isLoggedIn then
             local ped = PlayerPedId()
             if IsPedInAnyVehicle(ped, false) then
                 local veh = GetVehiclePedIsIn(ped, false)
+                local vehHash = GetEntityModel(veh)
                 local vehClass = GetVehicleClass(veh)
                 local speed = GetEntitySpeed(veh) * speedMultiplier
 
-                if vehClass ~= 13 and vehClass ~= 14 and vehClass ~= 15 and vehClass ~= 16 and vehClass ~= 21 then
+                if not IsWhitelistedVehClass(vehClass) and not IsWhitelistedVehicle(vehHash) then
```

Comment:
I'm concerned about loops within loops here. 
This will essentially loop over 2 tables for each game tick for as long as a player is in a vehicle.
I'm concerned about the potential performance impact of doing this